src/ml_models.py

```diff
@@ -422,6 +422,29 @@ def _create_loss_fn(self) -> Callable[[Tensor, Tensor], Tensor]:
         return nn.MSELoss()
 
 
+class Critic(nn.Module):
```

Comment:
when we're making other kinds of models, like for policies and samplers, we perform concatenation "outside" the NN module, which makes it easier to reuse general classes like Regressors, Classifiers, etc. can we do the same pattern here, and concatenate the state and action outside, and then use one of our existing ML models? this will also make sure we don't have issues with normalization, etc.